augly/image/functional.py

```diff
@@ -1969,7 +1969,7 @@ def random_noise(
     else:
         low_clip = 0.0
 
-    sigma = var ** 0.5
+    sigma = var**0.5
```

Comment:
Please remove all formatting changes like this from this PR.

Reply:
New commit should have solved this. Sorry, my bad 😅!